src/components/LocationCard/LocationCard.js

```diff
@@ -13,6 +13,7 @@ export const LocationCard = (props) => {
         As,
         locationId,
         locationTitle,
+        hasRadiusSlider,
         distanceRadius,
```

Comment:
![Screenshot 2020-05-18 at 13 01 14](https://user-images.githubusercontent.com/19436304/82206199-14719e80-9908-11ea-93c9-b0263aeb9e57.png)
The height of the input is not consistent with other inputs